app/views/course/material/folders/_folder.slim

```diff
@@ -1,8 +1,9 @@
 = content_tag_for(:tr, folder)
   th
     = link_to course_material_folder_path(current_course, folder) do
-      = fa_icon 'folder-open'.freeze
+      => fa_icon 'folder-open'.freeze
       => format_inline_text(folder.name)
+      => '(' + format_inline_text(folder.material_count + folder.children_count) + ')'
```

Comment:
Does Ruby's string interpolation work here? I think it should.

`"(#{format_inline_text(folder.material_count + folder.children_count)})"`